CountItems/utils/utils.js

```diff
@@ -58,7 +59,11 @@ function consolidateDataMetrics(target, source) {
     resTarget.objectCount._nonCurrentRestoring += objectCount && objectCount._nonCurrentRestoring ? objectCount._nonCurrentRestoring : 0;
     resTarget.objectCount._nonCurrentRestored += objectCount && objectCount._nonCurrentRestored ? objectCount._nonCurrentRestored : 0;
 
-    // Current and NonCurrent are the total of all other metrics
+    resTarget.usedCapacity._inflightsPreScan += usedCapacity && usedCapacity._inflightsPreScan ? usedCapacity._inflightsPreScan : 0;
```

Comment:
nit:
```suggestion
    resTarget.usedCapacity._inflightsPreScan += usedCapacity?._inflightsPreScan || 0;
```

Reply:
The node version in s3utils is not recent enough unfortunately, AFAIR...

Reply:
😢